tools/Makefile

```diff
@@ -3,16 +3,25 @@ CFLAGS := -Wall -Wextra -pedantic -std=c99 -g -O2
 PROGRAMS := yaz0 makeromfs elf2rom mkldscript vtxdis
 ZAPD := ZAPD/ZAPD.out
 
-all: $(PROGRAMS)
-	cd ZAPD && $(MAKE)
+all: $(PROGRAMS) zapd ido
 
 clean:
 	$(RM) $(PROGRAMS) $(addsuffix .exe,$(PROGRAMS))
 	$(MAKE) -C ZAPD clean
 
+ido:
+	$(MAKE) -C ido_recomp
+
+zapd:
+	cd ZAPD && $(MAKE)
```

Comment:
I think since we do it above for ido, we should probaby `MAKE -c ZAPD` here as well